### PR TITLE
fix: change in-app docs links to docs.frappe.io/books

### PR DESCRIPTION
**Pull request by ianandhum**
[docs.frappebooks.com](https://docs.frappebooks.com) is no longer resolvable. This breaks many documentation links both in-app and on README.md. This PR updates those broken links to point to https://docs.frappe.io/books.

### CODE REVIEW

**Comment by ianandhum** [comment]
Also I just noticed another open PR https://github.com/frappe/books/pull/908 which aims to fix this issue. Open to combining these two or feel free to close this PR or whatever makes sense in this case.

File: README.md [hunk 1]
```diff
@@ -143,7 +141,6 @@ If you want to contribute code then you can fork this repo, make changes and rai
 
 - [Telegram Group](https://t.me/frappebooks): Used for discussions and decisions regarding everything Frappe Books.
 - [GitHub Discussions](https://github.com/frappe/books/discussions): Used for discussions around a specific topic.
-- [Frappe Books Blog](https://tech.frappebooks.com/): Sporadically updated dev blog regarding the development of this project.
```
Comment (ianandhum) [inline README.md:146]: Couldn't find equivalent for https://tech.frappebooks.com/ :confused:

Comment (rtdany10) [reply README.md:146]: Hi @ianandhum, I don't think we have an alternative for that currently.

**Comment by rtdany10** [comment]
@ianandhum The other PR has been merged, could you cross check and confirm if any other URL is to be corrected and update the PR if required?

**Comment by ianandhum** [comment]
Yes this can be closed now.

**Pull request closed without merging**